nservicebus/azure-service-bus/batching.md

```diff
@@ -12,16 +12,16 @@ reviewed: 2016-04-26
 
 ## Batching store access
 
-In order to increase messages throughput, the Azure Service Bus batches operations prior to writing to its internal store. This involves delaying write operations. When batching is enabled, the write operations are delayed by 20 ms. Read operations are not affected. This increases throughput.
+In order to increase messages throughput, the Azure Service Bus batches operations prior to writing to its internal store. This involves delaying write operations by 20 ms. Read operations are not affected.
 
 Batching store operations should be disabled for low throughput scenarios requiring low latency.
 
-The batching is enabled by default. In order to disable it, set `EnableBatching` to `false` on [queues](/nservicebus/azure-service-bus/configuration/configuration.md#Queues), [topics](/nservicebus/azure-service-bus/configuration/configuration.md#Topics), and [subscriptions](/nservicebus/azure-service-bus/configuration/configuration.md#Subscriptions).
+Batching is enabled by default. In order to disable it, set `EnableBatching` flag to `false` on [queues](/nservicebus/azure-service-bus/configuration/configuration.md#Queues), [topics](/nservicebus/azure-service-bus/configuration/configuration.md#Topics), and [subscriptions](/nservicebus/azure-service-bus/configuration/configuration.md#Subscriptions).
```

Comment:
Perhaps the paragraph should start with "Batching store access". Just batching could be ambiguous.